controllers/comments.controller.js

```diff
@@ -0,0 +1,21 @@
+const {
+  removeCommentById,
+  checkIfCommentExists,
+} = require("../models/comments.model");
+
+exports.deleteCommentById = (req, res, next) => {
+  const { comment_id } = req.params;
+
+  const promises = [
+    checkIfCommentExists(comment_id),
+    removeCommentById(comment_id),
+  ];
```

Comment:
You might run into issues with this in future tests. These are both asynchronous functions and theres no guarantee you will always check that comment exists before you remove it. I would suggest instead just having one delete query and use RETURNING * to see if the comment existed from your delete query